tests/end-to-end/external-solution-integration/pkg/testsuite/send_event_to_compatibility_layer_and_check_eventId.go

```diff
@@ -0,0 +1,92 @@
+package testsuite
+
+import (
+	"fmt"
+	"time"
+
+	retrygo "github.com/avast/retry-go"
+	"github.com/kyma-project/kyma/tests/end-to-end/external-solution-integration/internal/example_schema"
+	"github.com/kyma-project/kyma/tests/end-to-end/external-solution-integration/pkg/retry"
+	"github.com/kyma-project/kyma/tests/end-to-end/external-solution-integration/pkg/step"
+	"github.com/kyma-project/kyma/tests/end-to-end/external-solution-integration/pkg/testkit"
+	"github.com/sirupsen/logrus"
+)
+
+// SendEventToCompatibilityLayerAndCheckEventId is a step which sends an event and checks if the correct EventId has been received
+type SendEventToCompatibilityLayerAndCheckEventId struct {
+	state       SendEventState
+	appName     string
+	payload     string
+	testService *testkit.TestService
+	retryOpts   []retrygo.Option
+}
+
+var counterLayer = 0
```

Comment:
Does it make sense to move this inside the `SendEventToCompatibilityLayerAndCheckEventId` struct definition?

Reply:
Fixed